README.md

```diff
@@ -1,5 +1,80 @@
-# Aapt2 package build workflow
+This repo is based on the work of lzhiyong at https://github.com/lzhiyong/android-sdk-tools and the modification of aapt by IBotPeaches for apktool: https://github.com/aosp-mirror/platform_frameworks_base/compare/main...iBotPeaches:platform_frameworks_base:apktool_14.0.0
 
-[![build workflow](https://github.com/revanced/aapt2/actions/workflows/build.yml/badge.svg)](https://github.com/revanced/aapt2/actions/workflows/build.yml)
+Building platform-tools and build-tools for Android, such as `aapt aapt2 aidl zipalign adb fastboot` ... etc.
 
-This repository contains a workflow to build the aapt2 binaries for Android.
+If you need other tools, please refer to existing tools to add CMake file.
+
+Currently, only aarch64 has been tested, and it is not guaranteed to work for other architectures.
+
+In order to save storage and bandwidth, I have removed all submodules and no longer provide source code compressed packages.
```

Comment:
You can clone with depth 0 to only fetch heads instead of the full history. The repo should check sources in VCS. Fetching sources is a git job not the build systems.